.github/workflows/publish.yml

```diff
@@ -3,6 +3,7 @@ name: Publish
 on:
   workflow_run:
     workflows: ["ReleaseStart"]
+    workflow_dispatch:
```

Comment:
I added `workflow_dispatch` to run the publication scripts manually, just in case the publication script failed we can try re-running easily instead of creating a new release on GitHub.